client/css/style.css

```diff
@@ -606,19 +609,30 @@ background on hover (unless active) */
 }
 
 /* Remove background on hovered/active channel when sorting/drag-and-dropping */
-#sidebar .ui-sortable-helper .chan.active, /* Networks */
-#sidebar .ui-sortable-helper .chan:hover,
-#sidebar .chan.ui-sortable-helper.active, /* Channels */
-#sidebar .chan.ui-sortable-helper:hover {
-	background-color: transparent;
+.ui-sortable-dragged,
+.ui-sortable-dragged .chan,
+#sidebar .ui-sortable-active .chan {
+	background: transparent !important;
```

Comment:
Whoa, the big guns :)
Is there a way to make the selector more specific to avoid `!important` at all?

Reply:
Fixed up.